### PR TITLE
[STRATCONN-1660] Add Limited Data Usage Mapping for FB CAPI Actions

### PR DESCRIPTION
**Pull request by rvadera12**
This PR adds three new fields {`Data Processing Options`, `Data Processing Country`, `Data Processing State`} to all `Facebook Conversions API` Actions . More information attached to the testing doc below. 

## Testing

_Tested in stage via deploying to stage.[Testing Doc](https://paper.dropbox.com/doc/FB-Conversions-API-Actions-LDU-Testing--BptaTXPyxYpNSCXud4Da0q0PAg-NEfw3v0hD8aALT0BBKCzr)._

- [ x] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [ x] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ x] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: packages/destination-actions/src/destinations/facebook-conversions-api/viewContent/index.ts [hunk 1]
```diff
@@ -84,6 +90,13 @@ const action: ActionDefinition<Settings, Payload> = {
       if (err) throw err
     }
 
+    let data_options, country_code, state_code
```
Comment (nick-Ag) [inline packages/destination-actions/src/destinations/facebook-conversions-api/viewContent/index.ts:93]: I feel like this repeated if block should be factored out into a single function somewhere if you agree

File: packages/destination-actions/src/destinations/facebook-conversions-api/initiateCheckout/index.ts [hunk 2]
```diff
@@ -85,6 +92,13 @@ const action: ActionDefinition<Settings, Payload> = {
       if (err) throw err
     }
 
+    let data_options, country_code, state_code
+    if (payload.data_processing_options) {
+      ;[data_options, country_code, state_code] = dataProcessingOptions(
```
Comment (rvadera12) [inline packages/destination-actions/src/destinations/facebook-conversions-api/initiateCheckout/index.ts:97]: @dlasky

File: packages/destination-actions/src/destinations/facebook-conversions-api/fb-capi-properties.ts [hunk 3]
```diff
@@ -111,6 +141,17 @@ export const validateContents = (contents: Content[]): IntegrationError | false
   return false
 }
 
+export const dataProcessingOptions = (
```
Comment (rvadera12) [inline packages/destination-actions/src/destinations/facebook-conversions-api/fb-capi-properties.ts:144]: @dlasky

**Review by nick-Ag — APPROVED** [review]
LGTM 🚀

**Pull request merged**